src/lib/services/user-service.ts

```diff
@@ -62,6 +63,7 @@ export interface ILoginUserRequest {
 }
 
 const saltRounds = 10;
+const disallowNPreviousPasswords = 5;
```

Comment:
Should this be configurable instead of a constant?

Reply:
I think for now we do the same as we do with salt rounds, just leave it as a constant